.infra/application.properties

```diff
@@ -27,4 +27,7 @@ debezium.transforms.ReadOperationFilter.condition=!(valueSchema.field('op') && v
 debezium.transforms.PostsFilter.type=io.debezium.transforms.Filter
 debezium.transforms.PostsFilter.language=jsr223.groovy
 debezium.transforms.PostsFilter.condition=!(valueSchema.field('op') && value.op == 'u' && value.source.table == 'post' && value.before.views != value.after.views)
+debezium.transforms.SourceFilter.type=io.debezium.transforms.Filter
+debezium.transforms.SourceFilter.language=jsr223.groovy
+debezium.transforms.SourceFilter.condition=!(valueSchema.field('op') && value.op == 'u' && value.source.table == 'source' && value.before.flags != value.after.flags)
```

Comment:
Just hopping `flags` here does not become JSON object, can be check locally I guess or just YOLO. 🚀

Reply:
I'm 99% sure debezium works based on regex flow anyway.
But let's see as logs still on now?

Reply:
Actually it missed events where flags is changed but something else also changed